From 9cd73c1a4e7ddd6719e5904b0b135e86b1b741cb Mon Sep 17 00:00:00 2001
From: Hans-Christoph Steiner <hans@eds.org>
Date: Tue, 8 Jul 2014 17:15:42 -0400
Subject: [PATCH 1/1] update suites with latest ubuntu releases

The ubuntu suites listed are quite old.  This updates to include all of the
releases currently named.  This patch is used in Lil' Debi, the Android
chroot app that is based on cdebootstrap.
---
 config/suites | 28 ++++++++++++++++++++++++++++
 1 file changed, 28 insertions(+)

diff --git a/config/suites b/config/suites
index 0708f4c..b738a7a 100644
--- a/config/suites
+++ b/config/suites
@@ -50,6 +50,34 @@ Suite: natty
 Config: generic-ubuntu
 Keyring: ubuntu-archive-keyring.gpg
 
+Suite: oneiric
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: precise
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: quantal
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: raring
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: saucy
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: trusty
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
+Suite: utopic
+Config: generic-ubuntu
+Keyring: ubuntu-archive-keyring.gpg
+
 Suite: oldstable
 Keyring: debian-archive-keyring.gpg
 
-- 
1.9.1

